docs/mobile/authentication.md

```diff
@@ -0,0 +1,7 @@
+## Mobile Playbook: Authentication
+
+#### Use JSON Web Tokens
+
+#### Support token invalidation
+
+#### Using social authentication providers (OAuth, OpenID)
```

Comment:
Maybe add a section on how to integrate with on device switch to app auth, w/ react native libs